packages/typegpu/src/core/pipeline/computePipeline.ts

```diff
@@ -153,6 +153,8 @@ class ComputePipelineCore {
             });
             return '';
           },
+
+          toString: () => `compute pipeline code resolution: ${this._entryFn}`,
```

Comment:
The pipelines seem to house labels as well, we can use them here.
The used entry function will show up in the resolution stack trace anyway.
```suggestion
          toString: () => `computePipeline:${this.label ?? '<unnamed>'}`,
```